src/figuresClasses.ts

```diff
@@ -2,10 +1,91 @@
+export interface Figure {
+  shape: `triangle` | `circle` | `rectangle`;
+  color: `red` | `green` | `blue`;
+  getArea: () => number;
+}
+
+export class Triangle implements Figure {
+  shape: `triangle`;
+
+  constructor(
+    public color: `red` | `green` | `blue`,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    this.shape = `triangle`;
```

Comment:
[CODE STYLE] The `shape` property is already declared in the `Figure` interface, and its value for the `Triangle` class is always `'triangle'`. You can remove the assignment in the constructor and declare the property as readonly at the class level.